### PR TITLE
roadmap: update roadmap

### PR DESCRIPTION
**Pull request by xiang90**
/cc @gyuho @heyitsanthony @fanminshi

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -6,26 +6,18 @@ This document defines a high level roadmap for etcd development.
...
+### etcd 3.2 (2017-May)
 - Stable scalable proxy
 - Proxy-as-client interface passthrough
 - Lock service
 - Namespacing proxy
-- JWT token based authentication
+- TLS Command Name and JWT token based authentication
 - Read-modify-write V3 Put
 - Improved watch performance
+- Support concurrent read
```
Comment (gyuho) [inline ROADMAP.md]: Is that for reducing lock contentions around mvcc?

**Comment by gyuho** [comment]
LGTM. Defer to @heyitsanthony

File: ROADMAP.md (hunk 1 above)
Comment (xiang90) [inline ROADMAP.md]: yes.

Comment (heyitsanthony) [inline ROADMAP.md]: `non-blocking concurrent reads`?

File: ROADMAP.md [hunk 2]
```diff
@@ -6,26 +6,18 @@ This document defines a high level roadmap for etcd development.
...
 
-### etcd 3.3 (?)
+### etcd 3.3 (??)
```
Comment (heyitsanthony) [inline ROADMAP.md]: 3.3, double mystery?

Comment (xiang90) [inline ROADMAP.md]: haha. no idea why i added another question mark... will remove...

**Comment by xiang90** [comment]
@heyitsanthony All fixed. PTAL.

**Comment by heyitsanthony** [comment]
lgtm

**Comment by fanminshi** [comment]
lgtm?

**Pull request merged**